app/uk/gov/hmrc/tradergoodsprofilesdatastore/config/DataStoreAppConfig.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.tradergoodsprofilesdatastore.config
+
+import com.google.inject.{Inject, Singleton}
+import play.api.Configuration
+import uk.gov.hmrc.tradergoodsprofilesdatastore.models.EnrolmentConfig
+@Singleton
+class DataStoreAppConfig @Inject() (configuration: Configuration) {
+
+  val host: String = configuration.get[String]("host")
```

Comment:
Do we need app name like was in the original config file? 

val appName: String = config.get[String]("appName")

Reply:
Yeah, noticed this. Perhaps, appName is not being used anywhere in the file even in front-end config. So removed it here.